io.github.rimsort.RimSort.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.rimsort.rimsort</id>
+
+  <name>RimSort</name>
+  <summary>  RimSort is an open source mod manager for the video game RimWorld. There is support for Linux, Mac, and Windows, built from the ground up to be a reliable, community-managed alternative to RimPy Mod Manager. </summary>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
```

Comment:
this is incompatible with the proprietary SDK license.

Reply:
What license shoud i use then ? because Rimsort github is AGPL but the flatpak must include propryetary sdk ?